UT4MasterServer.Services/Scoped/AccountService.cs

```diff
@@ -47,10 +59,35 @@ public async Task CreateAccountAsync(string username, string email, string passw
 		return await cursor.SingleOrDefaultAsync();
 	}
 
-	public async Task<IEnumerable<Account>> SearchAccountsAsync(string usernameQuery)
+	public async Task<PagedResponse<Account>> SearchAccountsAsync(string usernameQuery, AccountFlags flagsMask = AccountFlags.All, int skip = 0, int limit = 50)
 	{
-		var cursor = await accountCollection.FindAsync(account => account.Username.ToLower().Contains(usernameQuery.ToLower()));
-		return await cursor.ToListAsync();
+		var f = Builders<Account>.Filter;
```

Comment:
Unused variable?

Reply:
Odd, visual studio didnt throw any warning about this